pkg/lambdaworks/lambdaworks.go

```diff
@@ -159,3 +161,44 @@ func (a Felt) Div(b Felt) Felt {
 	C.lw_div(&a_c[0], &b_c[0], &result[0])
 	return fromC(result)
 }
+
+// Returns the number of bits needed to represent the felt
+func (a Felt) Bits() C.limb_t {
+	if a.IsZero() {
+		return 0
+	}
+	var a_c = a.toC()
+	return C.bits(&a_c[0])
+}
+
+func (a Felt) And(b Felt) Felt {
+	var result C.felt_t
+	var a_c C.felt_t = a.toC()
+	var b_c C.felt_t = b.toC()
+	C.felt_and(&a_c[0], &b_c[0], &result[0])
+	return fromC(result)
+}
+
+func (a Felt) Xor(b Felt) Felt {
+	var result C.felt_t
+	var a_c C.felt_t = a.toC()
+	var b_c C.felt_t = b.toC()
+	C.felt_xor(&a_c[0], &b_c[0], &result[0])
+	return fromC(result)
+}
+
+func (a Felt) Or(b Felt) Felt {
+	var result C.felt_t
+	var a_c C.felt_t = a.toC()
+	var b_c C.felt_t = b.toC()
+	C.felt_or(&a_c[0], &b_c[0], &result[0])
+	return fromC(result)
+}
+
+func (a Felt) Pow(b Felt) Felt {
+	var result C.felt_t
+	var a_c C.felt_t = a.toC()
+	var b_c C.felt_t = b.toC()
+	C.felt_pow(&a_c[0], &b_c[0], &result[0])
+	return fromC(result)
+}
```

Comment:
I don't understand why this function is needed for this builtin

Reply:
is not, there was a confusion with the ^ operand that i though that it was exponentiation but it was xor. i left it because maybe will be used in the future

Reply:
If this function is needed, please move this and its tests to a new PR

Reply:
Done here: #141